configs/frags/base.config

```diff
@@ -41,3 +41,7 @@ BR2_ENABLE_LOCALE_WHITELIST="C en_US"
 
 # fs utils
 BR2_PACKAGE_E2FSPROGS=y
+
+# udev needed for generating /dev/disk/by-uuid
+BR2_TOOLCHAIN_BUILDROOT_WCHAR=y
+BR2_PACKAGE_EUDEV=y
```

Comment:
From the config file:
https://storage.staging.kernelci.org/images/rootfs/buildroot/buildroot-baseline/20220322.1/x86/config

`BR2_PACKAGE_EUDEV=y` is there but not `BR2_TOOLCHAIN_BUILDROOT_WCHAR=y`.  Maybe it depends on some other configs?

Reply:
seems it was renamed to BR2_USE_WCHAR, I will update the PR